src/languages/en.js

```diff
@@ -264,6 +264,7 @@ export default {
         removePhoto: 'Remove photo',
         editImage: 'Edit photo',
         imageUploadFailed: 'Image upload failed',
+        deleteWorkspaceError: 'Sorry, there was an unexpected problem deleting your workspace avatar.',
```

Comment:
NAB: I was told to get a copy check for a similar error message. [Here](https://github.com/Expensify/Expensify/issues/219957#issuecomment-1222177053) is where that conversation is happening.